docs/mddocs/Quickstart/npu_quickstart.md

```diff
@@ -38,6 +38,16 @@ To update driver for Intel NPU:
    - Choose **Browse my computer for drivers**, navigate to the folder where you extracted the driver zip file, and select **Next**
    - Wait for the installation finished
 
+3. Uninstall and Install the driver [Optional]
```

Comment:
We may put `(Optional)` at the beginning of this sentence (as other QuickStart did for optional steps), and maybe change the description as following?

3. (Optional) Uninstall the current driver and install the newly downloaded one

Reply:
> We may put `(Optional)` at the beginning of this sentence (as other QuickStart did for optional steps), and maybe change the description as following?
> 
> 3. (Optional) Uninstall the current driver and install the newly downloaded one

Sure, have updated : )